### ISSUE TITLE
Bamboo

### ISSUE THREAD

**Issue by DragonTamr34** [danikk14g/Craftologia#8]
I created a new world to take advantage of the new ore gen systems. Unfortunately I need bamboo for taps and the glider ,but I cannot find any bamboo forests.  I cheated the biome finder and even added Nature's Compass but still could not find one. Do they spawn or is there another way to get bamboo? If they do and I just got really unlucky, is the only way to get bamboo by cheating it in?
Thanks,
DragonTamr34

**Comment by dancc7t** [comment]
a few days ago i found some isolated bamboos, but yeah i agree they are very rarer to find, so i will create an oredict to allow the use of bamboo or river canes in those craftings, will also try to either increase the chance of bamboo forest or allow bamboos to grow in other biomes using BoP config.

Did you try BoP Biome Finder with BambooForest essence?

**Comment by DragonTamr34** [comment]
Yes, I used the BoP Biome Finder with the proper essence. The device searched and then said there were not any close enough. The radius is about 14k blocks or so. So I teleported around and still could not find one. I even used the Natures compass so that I could tp to the biome, and there still was no Bamboo Forest. I guess I just got very unlucky with my biome generation. Also, where do diamonds spawn, the bedrock is not flat.

**Comment by ArnauGames** [comment]
You can obtain it cutting a long wood stick with a saw, the saw can be made with quertz, a quartz hammer and a iron file
